app/controllers/task_list_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TaskListController < ApplicationController
+  # TASK_LIST = [
+  #   {id: 1, status: false, action: "Master Rails"},
+  #   {id: 2, status: false, action: "Learn how to cook palak paneer"},
+  #   {id: 3, status: false, action: "Laundry"},
+  #   {id: 4, status: false, action: "Go for a walk"}
+  # ]
+
+  def index
+    # @task_list = TASK_LIST
+    @tasks = Task.order(:id)
+
+    # @tasks.sort_by { |task| Date.strptime(task.completion_date, "%m/%d/%Y") }
+
+    # @tasks = Task.order(Date.strptime(:completion_date, "%m/%d/%Y"))
+  end
+
+  def show
+
+    @task = Task.find( params[:id].to_i )
```

Comment:
It's a good idea to have some sort of if statement here to handle the occasions where you can't find the given task.  This applies to the actions, `edit`, `delete`, etc.